src/canvas/constants.js

```diff
@@ -67,6 +67,8 @@ export const COLORS = Object.freeze({
   PRIORITY_BACKGROUND: '#ededf0',
   PRIORITY_BORDER: '#d7d7db',
   PRIORITY_LABEL: '#272727',
+  USER_TIMING: '#45a1ff',
```

Comment:
We should use different colors for user timing marks, maybe grey or something that looks 'passive'.

What do you think?

Reply:
Yeah, something that doesn't upstage the React events/measures

Reply:
I'll update colors in my accessibility PR. We can get this merged in like this for now.